src/meta/client/src/grpc_client.rs

```diff
@@ -542,9 +544,12 @@ impl MetaGrpcClient {
                     .await;
                     match new_token {
                         Ok(token) => {
-                            return Ok(MetaServiceClient::with_interceptor(c, AuthInterceptor {
-                                token,
-                            }));
+                            let client =
+                                MetaServiceClient::with_interceptor(c, AuthInterceptor { token })
+                                    .max_decoding_message_size(16 * 1024 * 1024)
```

Comment:
The default limit is `4G`, do you want to make it smaller?

Reply:
The encoding limit is the max size a client can send and is 4GB, the decoding limit is the max size a server can receive and is 4MB.